src/main/archetype/ui.frontend.angular/README.md

```diff
@@ -40,3 +40,134 @@ Furthermore, an AEM ClientLib is generated from the app using the [`aem-clientli
 ${hash}${hash} Browser Support
 
 By default, this project uses [Browserslist](https://github.com/browserslist/browserslist)'s `defaults` option to identify target browsers. Additionally, it includes polyfills for modern language features to support older browsers (e.g. Internet Explorer 11). If supporting such browsers isn't a requirement, the polyfill dependencies and imports can be removed.
+
+
+
+#if ( $enableSSR == "y" )
+
+${hash} Frontend Serverless Build (Adobe IO Runtime)
+
+${hash}${hash} Features
+
+* AdobeIO Runtime (Serverless) rendering of the SPA editor 
+* Local + Remote development
+* Uses AIO CLI
+
+${hash}${hash} Installation
+
+1. Install [NodeJS](https://nodejs.org/en/download/) (v10+), globally. This will also install `npm`.
+2. Navigate to `ui.frontend` in your project and run `npm install`. (You must have run the archetype with `-DfrontendModule=angular` for this to work)
+3. If running locally the following images must be installed:
+   * docker pull openwhisk/action-nodejs-v10:latest
+   * docker pull adobeapiplatform/adobe-action-nodejs-v10:3.0.21
+
+${hash}${hash} Usage
+
+The following npm scripts are available:
+
+* `npm run build-serverless` - Build the serverless code
+* `npm run build-server` - Build express code (debuggable with sourcemaps)
+* `npm run start-ssr-express` - Runs the express code (debuggable with sourcemaps)
+* `npm run start-ssr-ioruntime` - Deploys the serverless code to a local docker container with AIO CLI. Requires docker running. Enables you to debug with visual studio code.
```

Comment:
Based on the feedback above I take it the runtime run scripts also don't apply?

Reply:
updated